src/python/bot/tasks/minimize_task.py

```diff
@@ -1047,19 +1047,22 @@ def do_js_minimization(test_function, get_temp_file, data, deadline, threads,
   data = do_line_minimization(test_function, get_temp_file, data, deadline,
                               threads, cleanup_interval, delete_temp_files)
 
-  tokenizers = [js_tokenizer.paren_tokenizer, js_tokenizer.comma_tokenizer]
-  for tokenizer in tokenizers:
-    current_minimizer = js_minimizer.JSMinimizer(
-        test_function,
-        max_threads=threads,
-        deadline=deadline,
-        cleanup_function=process_handler.cleanup_stale_processes,
-        single_thread_cleanup_interval=cleanup_interval,
-        get_temp_file=get_temp_file,
-        delete_temp_files=delete_temp_files,
-        tokenizer=tokenizer,
-        token_combiner=html_tokenizer.combine_tokens,
-        progress_report_function=functools.partial(logs.log))
+  tokenizer = AntlrTokenizer(JavaScriptLexer)
+
+  current_minimizer = js_minimizer.JSMinimizer(
```

Comment:
does it work better than the old minimizer? Some early experiments by @mpherman2 were showing that the old one worked better on real inputs, wondering how is it performing now

Reply:
It is performing much better now. We were able to get the real inputs minimized the same as the old minimizer, and the new tokenizer and changes to the minimizer should make minimization more reliable.

Reply:
Nice!!